src/librustc_mir/build/matches/test.rs

```diff
@@ -20,54 +20,72 @@ use syntax_pos::symbol::sym;
 
 use std::cmp::Ordering;
 
+struct TestTargetBlockBuilder<'tcx, 'a, 'b, 'c> {
+    test: &'a Test<'tcx>,
+    source_info: &'a SourceInfo,
```

Comment:
All of the fields apart from test here should be stored by value.

Reply:
Good point!